php74.spec

```
@@ -1706,6 +1706,9 @@ exit 0
 
 
 %changelog
+* Thu Jun 10 2022 Peter Zelina <pzelina@qualityunit.com> - 7.4.30-1
```

Can you please correct this date?
```suggestion
* Fri Jun 10 2022 Peter Zelina <pzelina@qualityunit.com> - 7.4.30-1
```